zerver/lib/actions.py

```diff
@@ -1470,6 +1471,12 @@ def do_send_messages(messages_maybe_none: Sequence[Optional[MutableMapping[str,
         '''
         user_ids = message['active_user_ids'] | set(user_flags.keys())
 
+        def get_user_group_id(user_id: int, message: Message) -> Optional[int]:
+            if message.mentions_user_ids.get(user_id, None) is not True:
```

Comment:
`is not True` is a weird way to phrase this; why not just `if not blah.get(user_id, False)`?